tests/test_readFitsCatalog.py

```diff
@@ -118,15 +124,25 @@ def testHDU1GivenNames(self):
         for inname, outname in zip(self.arr1.dtype.names, arr.dtype.names):
             des_outname = column_map.get(inname, inname)
             self.assertEqual(outname, des_outname)
-            self.assertTrue(np.array_equal(self.arr1[inname], arr[outname]))
+            if inname == 'name':
+                for el1, el2 in zip(self.arr1[inname], arr[outname]):
+                    self.assertTrue(el1, el2.decode())
+            else:
+                self.assertTrue(np.array_equal(self.arr1[inname], arr[outname]))
 
     def testHDU2(self):
         """Test reading HDU 2 with original order"""
         config = ReadFitsCatalogTask.ConfigClass()
         config.hdu = 2
         task = ReadFitsCatalogTask(config=config)
         arr = task.run(FitsPath)
-        self.assertTrue(np.array_equal(arr, self.arr2))
+        for name in self.arr1.dtype.names:
+            # Table form disk ends up being bytes and in memory is unicode
```

Comment:
I think this should be "from disk".  But I'm confused why there are these two options below?  Why isn't it one or the other?

Reply:
I'll fix the typo.

I don't know why it does this, but the array constructed from a Table read from a fits file on disk has the strings as type `bytes`.  If the array comes from a Table constructed from a FITS object in memory, they are `unicode` even if I try to force them to be `bytes`.

I didn't want to try to figure out if it was a bug or not, so just handle the one string like column separately.  I know that's not particularly satisfying, but I'd prefer to figure out the difference without the pressure of the release blockage.  I'll file a ticket if that's ok with you.

Reply:
At least put that in the comments, though is it possible that this could bite a user?

Reply:
Hold on ... what ticket is this associated with?  It should be in the PR name.

Reply:
Gah.  Very sorry about that.  I was in a rush to make the PRs.  This is DM-18036.